### PR TITLE
Updated the styles of the SwiftMailer commands

### PR DESCRIPTION
**Pull request by javiereguiluz**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | -
| License       | MIT
| Doc PR        | -

swiftmailer_1

swiftmailer_2

swiftmailer_3

swiftmailer_4

### CODE REVIEW

**Comment by stof** [comment]
Dropping support for Symfony 2.3 seems a bad idea. Core bundles need to keep support for the LTS. And it would be great to avoid having to maintain 2 versions of the bundle in parallel.

File: Command/DebugCommand.php [hunk 1]
```diff
@@ -117,30 +101,35 @@ protected function outputMailer(OutputInterface $output, $name)
...
+        $tableHeaders = array('Property', 'Value');
+        $tableRows = array();
+
         $transport  = $mailer->getTransport();
         $spool = $this->getContainer()->getParameter(sprintf('swiftmailer.mailer.%s.spool.enabled', $name)) ? 'YES' : 'NO';
         $delivery = $this->getContainer()->getParameter(sprintf('swiftmailer.mailer.%s.delivery.enabled', $name)) ? 'YES' : 'NO';
         $singleAddress = $this->getContainer()->getParameter(sprintf('swiftmailer.mailer.%s.single_address', $name));
 
-        $output->writeln($this->getHelper('formatter')->formatSection('swiftmailer', sprintf('Mailer "%s"', $name)));
+        $output->title(sprintf('Configuration of the Mailer "%s"', $name));
```
Comment (stof) [inline Command/DebugCommand.php]: this triggers a fatal error if ``$output`` is null, which is allowed in your method signature.

File: Command/DebugCommand.php [hunk 2]
```diff
@@ -52,24 +53,23 @@ protected function configure()
      */
     protected function execute(InputInterface $input, OutputInterface $output)
     {
+        $stdout = $output;
```
Comment (stof) [inline Command/DebugCommand.php]: IMO, keeping ``$output`` ofr the raw output and changing the name for the styled output would be more readable if you deal with both.

**Comment by javiereguiluz** [comment]
@stof I agree with you. Let's block this pull request for now and let's merge it on May 2016 when Symfony 2.3 will die.

File: Command/SendEmailCommand.php [hunk 3]
```diff
@@ -39,7 +40,7 @@ protected function configure()
             ->setHelp(<<<EOF
 The <info>swiftmailer:spool:send</info> command sends all emails from the spool.
 
-<info>php %command.full_name% --message-limit=10 --time-limit=10 --recover-timeout=900 --mailer=default</info>
+<info>php app/console swiftmailer:spool:send --message-limit=10 --time-limit=10 --recover-timeout=900 --mailer=default</info>
```
Comment (Koc) [inline Command/SendEmailCommand.php]: Why thic changed?

Comment (stof) [inline Command/SendEmailCommand.php]: -1 for that btw. Using the placeholders is better, as it avoids having to maintain the command name in multiple places. Thus, the console file is not in ``app`` anymore in the Symfony 3 structure, meaning that your updated help message is wrong on Symfony 3.

**Comment by fabpot** [comment]
@javiereguiluz Can you update this PR? I'd like to drop support for 2.3 now that it's not maintained anymore. Thanks.

**Comment by fabpot** [comment]
@javiereguiluz You were faster than I expected :) I've just bumped version to 2.4 which drops support for 2.3. I've also added a CHANGELOG. Can you rebase to again and add an entry in the new CHANGELOG?

**Comment by fabpot** [comment]
@javiereguiluz Can you have a look at the broken tests? Apparently, there is one var `$sent` that is used but not defined. Thanks.

**Comment by fabpot** [comment]
Thank you @javiereguiluz.

**Pull request closed without merging**